OG_Format.adoc

```diff
@@ -911,3 +862,58 @@ _Operational phase:_ 3 months – Oct 2022 to Dec 2023
 2 years after agreement on the Terms of Reference OG1.0 will become the unique format for the OceanGliders program.
 
 [[_heading=h.1egqt2p]]Glider missions not delivering OG1.0 will not be considered as part of the OceanGliders program. It will be encouraged that legacy files be converted and added to OceanGliders final repository
+
+[appendix]
+== Examples
+
+=== Platform, Programme, and Network
+
+Example 1:
+
+* platform (i.e. glider mission) = kraken_20210205
+* Programme = MOOSE glider program
+* Sites = MOOSE_T00, MOOSET_02
+* Networks = Mediterranean Ocean Observing Systems for the Environment (MOOSE), Boundary Ocean Observing Network (BOON), Water Transformation task team”
+
+Example 2:
+
+* platform : sdeep09_sdeep04_20200929
+* Programme : SOCIB Glider Programme
```

Comment:
Here ". : ." 

If we decide to use ":" each time the space before : should be removed

i.e. everywhere
```suggestion
* Programme: SOCIB Glider Programme
```